### DISCUSSION TITLE
Help with API

### DISCUSSION THREAD

**Discussion by colcyber** [lllyasviel/Fooocus#1652] [Q&A]
Hi,
i problem with API (Python), i'm not expert and i can't understand how can i use it.
Please, can you help me? I need simple example with easy standard prompt.
Thanks all

**Comment by mashb1t** [comment]
You can find a working example in https://github.com/lllyasviel/Fooocus/issues/172#issuecomment-1848709342. Hope this helps.

**Reply by colcyber** [reply]
Thanks, i think it work, i show "Loaded as API: ..."
but it don't create any image.
How work it?
thanks and sorry again

**Reply by mashb1t** [reply]
Please check again if you're using the latest Fooocus version and if the parameter count is the same as in http://127.0.0.1:7865/?view=api (fn 29 i assume)
